nav2_smac_planner/src/smac_planner_hybrid.cpp

```diff
@@ -223,6 +227,10 @@ void SmacPlannerHybrid::configure(
     _expansions_publisher = node->create_publisher<geometry_msgs::msg::PoseArray>("expansions", 1);
   }
 
+  if (_debug_visualizations) {
+    _planned_footprints_publisher = node->create_publisher<visualization_msgs::msg::MarkerArray>("planned_footprints", 1);
```

Comment:
eg the `_expansions_publisher` should be made under this if statement when `_viz_expansions` is gone.

Reply:
Because they're both debug visualizations